requirements.txt

```diff
@@ -1,2 +1,2 @@
 git+git://github.com/canonical/operator#egg=ops
-pymongo
+pymongo=3.12.1
```

Comment:
Hi, if `directConnection=True` with `MongoClient` fixes this issue, could you please updated pin to latest stable version, or remove pin if you prefer.